public/app.js

```diff
@@ -60,7 +60,7 @@ if (options.background.enabled) {
 	views.background.style.padding = '0px';
 }
 
-injectStylesheetForTheme(`./prism-themes/prism-${options.theme}.css`);
+injectStylesheetForTheme(`/prism-themes/prism-${options.theme}.css`);
```

Comment:
I think it's better practice to have `./prism-themes/prism-${options.theme}.css`, idk though. I didn't even know it worked without the `./`

Reply:
it doesn't work with ./